src/LinuxTracing/PerfEventOpen.h

```diff
@@ -107,6 +107,7 @@ static constexpr uint16_t SAMPLE_STACK_USER_SIZE_8BYTES = 8;
 // a lower value. For the current layout of perf_event_stack_sample_fixed, the maximum
 // size is 65312. We leave some extra room with our flag (see `ClientFlags.cpp`).
```

Comment:
Was this comment changed recently? "We leave some extra room with our flag (see `ClientFlags.cpp`)" doesn't make a lot of sense to me. The idea was to simply not have to count the exact bytes: we have to be lower than `(1u << 16u) - 8`, let's just pick a low enough value.